src/plugin/yjsEditor.ts

```diff
@@ -3,11 +3,11 @@ import invariant from 'tiny-invariant';
 import * as Y from 'yjs';
 import { applyYjsEvents } from '../applyToSlate';
 import applySlateOps from '../applyToYjs';
-import { SharedType } from '../model';
-import { toSlateDoc } from '../utils/convert';
+import { SharedType, slateYjs } from '../model';
+import { toSlateDoc } from '../utils';
 
 const IS_REMOTE: WeakSet<Editor> = new WeakSet();
-const IS_LOCAL: WeakSet<Editor> = new WeakSet();
+const LOCAL_OPERATIONS: WeakMap<Editor, Set<Operation>> = new WeakMap();
```

Comment:
```suggestion
const LOCAL_OPERATIONS: WeakMap<Editor, WeakSet<Operation>> = new WeakMap();
```

If we change the Set to a WeakSet we can get rid of the clear